ui/src/app/machines/views/MachineDetails/MachineTests/MachineTestsTable/TestActions/TestActions.tsx

```diff
@@ -19,8 +21,7 @@ const TestActions = ({ scriptResult, setExpanded }: Props): JSX.Element => {
   const location = useLocation();
   const canViewDetails = !scriptResultInProgress(scriptResult.status);
   const hasMetrics = scriptResult.results.length > 0;
-  // TODO - Update ContextualMenu props to be able to accept non Button props
-  const links: TSFixMe = [];
+  const links: ContextualMenuProps<DataTestElement<LinkProps>>["links"] = [];
```

Comment:
Getting a bit nested here 😬

Reply:
Thanks, I've moved the `DataTestElement<LinkProps>` to a type so it reads a bit better.